lib/thunderbird_common.pm

```diff
@@ -153,10 +158,23 @@ sub tb_send_message {
             assert_and_click "thunderbird_focus_security_exception-TW";
             assert_and_click "thunderbird_select_sentemail_window-TW";
         }
+
         assert_and_click "thunderbird_maximized_send-message";
+        if ($proto eq 'imap') {
+            #need wait few minutes to send message
+            wait_still_screen(stilltime => 580, timeout => 600, similarity_level => 45);
+            #this TB version need cancel and reply email to create sent email folder., maybe a bug?
+            assert_and_click "thunderbird_cancel_send-message";
+            assert_and_click "thunderbird_close_send-message";
+            assert_and_click "thunderbird_dont_save";
+            assert_and_click "thunderbird_select_email_reply";
+            assert_and_click "thunderbird_reply_email";
+            assert_and_click "thunderbird_maximized_send-message";
+        }
+        assert_screen('thunderbird_sent-folder-appeared', timeout => 180);
     }
     else {
-        assert_screen 'thunderbird_sent-folder-appeared';
+        assert_screen('thunderbird_sent-folder-appeared', 90);
```

Comment:
no need to add parentheses. Just write `assert_screen 'thunderbird_sent-folder-appeared', 90;`

Reply:
ok, done